### PR TITLE
BREAKING: Remove symbol from type

### LINKED ISSUE #39: Should FSA allow Symbol action types?
The Redux BDFL has [mentioned before that Symbols as action types is a bad idea](https://github.com/acdlite/redux-actions/issues/20#issuecomment-134340125).

[We're considering dropping support for Symbols in redux-actions](https://github.com/acdlite/redux-actions/issues/126), and wanted to cross-post that issue here since this is really about FSA shape.

Is anyone using Symbols as action types? What are the advantages over using strings? Standardizing on strings would lead to conceptual simplicity and potentially less special-casing.

### PR DESCRIPTION
**Pull request by unional**
Fix #39

### CODE REVIEW

**Comment by JaKXz** [comment]
This is something that should be reflected in the exported methods as well.

**Comment by JaKXz** [comment]
I believe this would also resolve #9, if there's general support for it.

File: src/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
 export function isFSA(action) {
   return (
     isPlainObject(action) &&
-    (isString(action.type) || isSymbol(action.type)) &&
```
Comment (JaKXz) [inline src/index.js]: There is an import that needs to be removed as well.

Comment (unional) [reply src/index.js]: Thanks. I was editing on github directly. Missed the import. 😛

**Comment by timche** [comment]
Looks good 👍

**Pull request merged**